test/smoke/package.json

```diff
@@ -22,7 +22,7 @@
     "gulp-untar": "^0.0.7",
     "gulp-vinyl-zip": "^2.1.2",
     "mkdirp": "^0.5.1",
-    "mocha": "^5.2.0",
+    "mocha": "^3.2.0",
```

Comment:
Why didn't you update package-lock.json?

Reply:
This tests using yarn, so I update `yarn.lock`